sim/paladin/holy_shield.go

```diff
@@ -0,0 +1,116 @@
+package paladin
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/stats"
+)
+
+var HolyShieldValues = []struct {
+	level    int32
+	spellID  int32
+	procID   int32
+	manaCost float64
+	damage   float64
+}{
+	{level: 30, spellID: 20925, procID: 20955, manaCost: 150, damage: 65},
+	{level: 50, spellID: 20927, procID: 20956, manaCost: 195, damage: 95},
+	{level: 60, spellID: 20928, procID: 20957, manaCost: 240, damage: 130},
+}
+
+func (paladin *Paladin) registerHolyShield() {
+
+	if !paladin.Talents.HolyShield {
+		return
+	}
+
+	var numCharges int32 = 4
```

Comment:
knit: This can be `numCharges := int32(4)`

Reply:
Done.